server/tournament.py

```diff
@@ -544,6 +576,50 @@ def update_tournament_spirit_rankings(tournament: Tournament) -> None:
     tournament.save()
 
 
+def check_if_authorised_to_submit_scores_by_match(match: Match, player: Player) -> tuple[bool, int]:
```

Comment:
Can this be called something like `can_submit_match_scores`?

Reply:
yep we can rename this... too long func name